src/pylexibank/commands/consonant_clusters.py

```diff
@@ -0,0 +1,60 @@
+"""
+Check for (potentially) problematic consonant clusters > 2.
+"""
+
+from clldutils.clilib import Table, add_format
+from cldfbench.cli_util import with_dataset, add_catalog_spec
+from pylexibank.cli_util import add_dataset_spec
+import collections
```

Comment:
Could we move imports from stdlib to the top, please?

Reply:
My bad, fixed :)